.eslintrc.js

```diff
@@ -18,6 +18,11 @@ module.exports = {
                 'rulesdir/no-multiple-api-calls': 'off',
                 'rulesdir/prefer-import-module-contents': 'off',
                 'no-constructor-return': 'off',
+                'max-classes-per-file': 'off',
+                'arrow-body-style': 'off',
+                'es/no-nullish-coalescing-operators': 'off',
+                'rulesdir/prefer-underscore-method': 'off',
+                'es/no-optional-chaining': 'off',
```

Comment:
NAB - why are we turning off these rules?

Reply:
These rules were connected to underscore library. Some migrations required using JS features blocked by ESlint to make them work the same

Reply:
We turned off some ESlint rules because in some cases we had to use some JS features that were blocked by them, to make functions work the same